frontend/src/components/activity/ButtonNestedContentNodeAdd.vue

```diff
@@ -126,3 +144,32 @@ export default {
   },
 }
 </script>
+
+<style scoped lang="scss">
+.ec-button-contentnode-add {
+  transition: all 0.2s linear;
+  transition-property: box-shadow, background-color;
+  box-shadow: inset 0 0 115px #f8fcff;
+  border: 2px dotted rgb(13, 71, 161, 0.36);
+  background-color: transparent !important;
+  justify-content: start;
+  &:hover {
+    outline: none;
+    border: 1px solid rgb(13, 71, 161, 0.6);
+    box-shadow: inset 0 0 115px #eef2fa;
+    padding: 1px 17px;
+  }
+}
+
+.dragging-content-node .ec-button-contentnode-add-wrapper--single {
+  position: absolute;
+  inset: 0;
+  margin-bottom: 6px !important;
+}
+
+.ec-button-contentnode-add-wrapper--center .ec-button-contentnode-add {
+  justify-content: center;
+  min-height: 36px;
+  height: 100% !important;
```

Comment:
i don't start to ask why the 3 !important here are needed.

Here a case where the height 100% might not be intended:
![grafik](https://github.com/ecamp/ecamp3/assets/1506818/1f894aca-7a53-4fc6-81f7-34df60731b23)

Reply:
It is exactly where the `100%` is intended 😄